src/Sylius/Bundle/ApiBundle/Doctrine/ORM/QueryExtension/Admin/TaxonImage/TaxonImageExtension.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\Doctrine\ORM\QueryExtension\Admin\TaxonImage;
+
+use ApiPlatform\Doctrine\Orm\Extension\QueryResultItemExtensionInterface;
+use ApiPlatform\Doctrine\Orm\Util\QueryNameGeneratorInterface;
+use ApiPlatform\Metadata\Operation;
+use ApiPlatform\Metadata\Post;
+use Doctrine\ORM\QueryBuilder;
+use Sylius\Component\Core\Model\TaxonImage;
+
+final class TaxonImageExtension implements QueryResultItemExtensionInterface
```

Comment:
Name it concerning the logic that is in this extension, or at least sth like `ResultExtension` as it is already in `TaxonImage` directory 🤔